docs/api.rst

```diff
@@ -22,3 +27,13 @@ series
 ------
 
 .. autoclass:: series
+
+signal
+------
+
+.. autoclass:: signal
```

Comment:
This should be `autofunction`. Interestingly, it does not fail this way, but it produces

![image](https://user-images.githubusercontent.com/173624/125251899-16f73a80-e2f8-11eb-8ef8-84930d244ae9.png)

Reply:
I copied and paste from `series`, so I will change it for `series`, too.

Reply:
done